resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -419,6 +421,179 @@ class Audio
     }
 }
 
+<# Notes:
+Text cursor indicator: (Colorful arrows above/below the cursor that blinks.)
+- Enabled: Empty string by default. Off is any value other than "cursorindicator". 
+- Size: Key doesn't exist by default. Values are literally "Size 1" to "Size 5"
+- Color "16711871" by default, which seems like 167, 118, 71. 
+
+Text cursor (Cursor that blinks.)
+- Thickness: 1 by default. Range: 1-20
+#>
+[DSCResource()]
+class TextCursor
+{
+    # Key required. Do not set.
+    [DscProperty(Key)] [string] $SID
+    [DscProperty()] [nullable[bool]] $TextCursorIndicatorStatus
+    [DscProperty()] [int] $TextCursorIndicatorSize
+    [DscProperty()] [int] $TextCursorIndicatorColor
+    [DscProperty()] [int] $TextCursorThickness

```

Comment:
It's already assumed that these properties relate to `TextCursor` so these can be shortened. 
```suggestion
    [DscProperty()] [nullable[bool]] $Status
    [DscProperty()] [int] $Size
    [DscProperty()] [int] $Color
    [DscProperty()] [int] $Thickness

```